docs/how-to-guides/software-management/software-support.md

```diff
@@ -1,34 +1,52 @@
+
+## Support Sprint Process 
+
+Support sprints are dedicated efforts, time-boxed to 3 weeks quarterly. We schedule these as part of our internal scheduling procedures, and support sprints are scheduled 3 to 6 weeks ahead of time. This includes a prioritization meeting where we go through the issues accross our repositories and put them in prioritization order. These meetings are open to the public. 
+
+??? Question "How do I get Notified of Support Sprints and prioritization meetings?" 
+    We email anyone who has opted in to our software support newsletter which includes when the support sprint was scheduled, when the prioritization meetings are happening, and a link to the meeting. These are sent shortly after it's scheduled and again right before the meeting. 
+    
+    Sign up at [here](https://public.govdelivery.com/accounts/USDOIGS/signup/39118). 
+
+### Step 1: Scheduling the Support Sprint
 While there is some leeway in the exact timing of each support sprint, the sprints must be scheduled quarterly and should be scheduled roughly 3 months apart. Project scheduling should be coordinated by the project/technical lead and the software lead during routine software scheduling meetings.
 
 After setting up the sprint schedule, the technical lead should create a software support board and communicate the schedule to contributors.
 
-## Step 2: Setting up the Project Board
+### Step 2: Setting up the Project Board
 Immediately after scheduling the support sprint, the technical lead should create a project board to facilitate the tracking, prioritization, and assignment of issues.  This has traditionally taken the form of a GitHub [projects board](https://github.com/orgs/DOI-USGS/teams/astrogeology-developers/projects) or a GitHub [discussion post](https://github.com/DOI-USGS/ISIS3/discussions), both of which provide automated tracking for issues that are hosted on GitHub. 
 
 !!! Note "Due to recent changes in permissions and organizational boundaries, project boards can only be created as private (within-organization), which precludes external contributors from viewing or editing the board."
 
-## Step 3: Notifying the Community
-With a project board in place, there is now a mechanism to store incoming issues, and the technical lead can notify the community of the upcoming sprint.  There is no "correct" way to notify community members, but the recommended method is via an MS Teams calendar event, which allows the host to send an email with an associated meeting invitation.
+### Step 3: Notifying the Community
+With a project board in place, there is now a mechanism to store incoming issues, and the technical lead can notify the community of the upcoming sprint.  A new GovDelivery bulletin. Bulletin are emails that get broadcasted to subscribers.  
```

Comment:
The last section reads like it is for the community. This section has jargon (what is a technical lead and why does someone external care?) and reads like it is an instruction for folks on the development team. Consider revising.

Reply:
This is kind of half for the community but something a dev can follow. I can move that to a collapsed admonition so it's out of the way.